src/test/operators/join_test_runner.cpp

```diff
@@ -585,6 +655,17 @@ TEST_P(JoinTestRunner, TestJoin) {
     print_configuration_info();
     FAIL();
   }
+
+  // TODO(anyone) check performance data if an index join was used.
```

Comment:
This todo is pretty concrete, why not implement it?

Reply:
Due to time constraints in the past. Is now being tackled.